### PR TITLE
feat(hedera): added demo TT file for hedera mainnet network : krypc

### PR DESCRIPTION
**Pull request by AnithaKrypC**
added demo TT file for hedera mainnet network

Updated test cases

### CODE REVIEW

File: src/test/endorsement-chain.spec.ts [hunk 1]
```diff
@@ -54,3 +54,30 @@ test("Endorsement chain title and actions are rendered correctly", async (t) =>
...
+  // await t.expect(EndorsementChainAddress2.count).eql(2);
+  //
+  // await t.expect(DocumentIssuedAction.count).eql(1);
+  //
+  // await t.expect(EndorseNomineeAction.count).eql(1);
+  // await t.expect(ChangeOwnershipAction.count).eql(1);
+  // await t.expect(TransferHoldershipAction.count).eql(1);
+  //
+  // await t.expect(SurrenderToIssuerAction.count).eql(1);
+  // await t.expect(SurrenderAcceptedAction.count).eql(1);
```
Comment (isaackps) [inline src/test/endorsement-chain.spec.ts]: why are all these test commented out?

Comment (AnithaKrypC) [reply src/test/endorsement-chain.spec.ts]: because of this issue
Screenshot from 2024-03-22 15-27-31

Comment (isaackps) [reply src/test/endorsement-chain.spec.ts]: it should be the RPC url that was used. can use 1 that can be queried and it should be using an archived node

**Comment by AnithaKrypC** [comment]
We updated RPC Url in configuration. Tested locally and attached screenshot also @isaackps 

image

**Comment by AnithaKrypC** [comment]
@isaackps could you please re-trigger the test once

**Comment by AnithaKrypC** [comment]
@isaackps could you please review and run the testcases

**Comment by MinHtet-O** [comment]
Hi @AnithaKrypC 

Can you please rebase this and add the following tests ( testcafe ) for hedera mainnet as well?

- Endorsement chain title and actions are rendered correctly for hederamainnet
- sample document is rendered correctly when DNS TXT is verified for hederamainnet

**Comment by AnithaKrypC** [comment]
> Hi @AnithaKrypC
> 
> Can you please rebase this and add the following tests ( testcafe ) for hedera mainnet as well?
> 
> * Endorsement chain title and actions are rendered correctly for hederamainnet
> * sample document is rendered correctly when DNS TXT is verified for hederamainnet

@MinHtet-O  sure will update

**Comment by rongquan1** [comment]
@AnithaKrypC 

It looks like this branch is out-of-date with the master branch. Could you please rebase it to include the latest changes?

**Comment by AnithaKrypC** [comment]
> @AnithaKrypC
> 
> It looks like this branch is out-of-date with the master branch. Could you please rebase it to include the latest changes?



> @AnithaKrypC
> 
> It looks like this branch is out-of-date with the master branch. Could you please rebase it to include the latest changes?

updated but still mainnet testcase is not passing through but we have tested in tradetrust.io/verify. it is fine there

**Comment by rongquan1** [comment]
@AnithaKrypC 

Please remove the test case for the Hedera Mainnet network, as our test environments are limited to the test network.

**Comment by AnithaKrypC** [comment]
> @AnithaKrypC
> 
> Please remove the test case for the Hedera Mainnet network, as our test environments are limited to the test network.

removed and committed now. @rongquan1

**Comment by rongquan1** [comment]
@AnithaKrypC 

Please review the [RPC URL.](https://hedera-mainnet-json-rpc.krypc.com/)

The 'Access-Control-Allow-Origin' header currently includes multiple values (*, *), but only one is allowed.

**Comment by AnithaKrypC** [comment]
> @AnithaKrypC
> 
> Please review the [RPC URL.](https://hedera-mainnet-json-rpc.krypc.com/)
> 
> The 'Access-Control-Allow-Origin' header currently includes multiple values (*, *), but only one is allowed.

Updated RPC url settings

**Comment by rongquan1** [comment]
@AnithaKrypC 

No 'Access-Control-Allow-Origin' header is present on the requested resource. 

It needs to include one valid value, but not multiple.

**Comment by AnithaKrypC** [comment]
> @AnithaKrypC
> 
> No 'Access-Control-Allow-Origin' header is present on the requested resource.
> 
> It needs to include one valid value, but not multiple.

we have added only this - Access-Control-Allow-Origin '*' 
this issue is only in mainnet RPC or testnet RPC as well?

**Comment by rongquan1** [comment]
@AnithaKrypC 

Thanks, its working now.

**Review by rongquan1 — APPROVED** [review]

**Comment by rongquan1** [comment]
@AnithaKrypC 

Do you also want to update the RPC URLs to the rest of the repos?

**Comment by AnithaKrypC** [comment]
> @AnithaKrypC
> 
> Do you also want to update the RPC URLs to the rest of the repos?

@rongquan1 will open PR in other repos

**Comment by rongquan1** [comment]
@AnithaKrypC
Can check if all the PRs have been opened yet, or are they still in progress?

**Pull request merged**